### PR TITLE
Add HALO Network mainnet

### PR DESCRIPTION
**Pull request by f4nzkrys31**
updated json file and filename

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Please make it valid json and use the correct filename

**Comment by f4nzkrys31** [comment]
Hi , may i check why the submitted mainnet still cant be search on chainlist.org.
do advice how do get listed on chainlist.org
thanks

**Comment by ligi** [comment]
it was never merged - so it is not there - and it will not get merged when there is a file `_data/chains/CAIP-2.json`

**Comment by f4nzkrys31** [comment]
Hi,thanks for advice. Has removed _data/chains/CAIP-2.json. The file will be automatically merged?

File: _data/chains/eip155-1280.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "HALO Mainnet",
+  "chain": "HALO",
+  "network": "mainnet",
+  "rpc": [
+    "https://https://nodes.halo.land",
```
Comment (ligi) [inline _data/chains/eip155-1280.json:6]:
```suggestion
    "https://https://nodes.halo.land"

```

File: _data/chains/eip155-1280.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "nativeCurrency": {
+    "name": "HALO",
+    "symbol": "HO",
+    "decimals": 18
+  },
+  "infoURL": "https://halo.land/#/",
+  "shortName": "HO",
+  "chainId": 1280,
+  "networkId": 1280,
+  "icon": "HALONetwork",
```
Comment (ligi) [inline _data/chains/eip155-1280.json:19]:
```suggestion
```

**Comment by f4nzkrys31** [comment]
Hi, do i need to amend anything?

**Comment by ligi** [comment]
yea - CI is not happy about: explorer standard must be 'none' or 'EIP3091'
also I removed the Icon for now - if you want it back you need to actually add the definition ;-)

**Comment by f4nzkrys31** [comment]
had removed the explorer and pending review.

**Comment by ligi** [comment]
you do not need to remove the whole explorer section - just add the `standard` field
and if you remove it - please remove it proper - you now produced invalid json

**Comment by f4nzkrys31** [comment]
> you do not need to remove the whole explorer section - just add the `standard` field and if you remove it - please remove it proper - you now produced invalid json

sorry for that. not a tech guy.
i hope it is correct now.

**Review by ligi — CHANGES_REQUESTED** [review]
seems this explorer is not EIP3091 compatible - e.g. see:

https://browser.halo.land/block/1

please fix or use 'none'

also when using a icon here you need to define it

**Comment by f4nzkrys31** [comment]
updated to "none"

File: _data/chains/eip155-1280.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "nativeCurrency": {
+    "name": "HALO",
+    "symbol": "HO",
+    "decimals": 18
+  },
+  "infoURL": "https://halo.land/#/",
+  "shortName": "HO",
+  "chainId": 1280,
+  "networkId": 1280,
+    "explorers": [{
```
Comment (ligi) [inline _data/chains/eip155-1280.json:19]:
```suggestion
"explorers": [{
```

File: _data/chains/eip155-1280.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  "nativeCurrency": {
+    "name": "HALO",
+    "symbol": "HO",
+    "decimals": 18
+  },
+  "infoURL": "https://halo.land/#/",
+  "shortName": "HO",
+  "chainId": 1280,
+  "networkId": 1280,
+"explorers": [{
```
Comment (ligi) [inline _data/chains/eip155-1280.json:19]:
```suggestion
  "explorers": [{
```

**Comment by ligi** [comment]
:tada:

**Pull request merged**